test/e2e/wallet-watcher.feature.ts

```diff
@@ -1,401 +1,54 @@
 import * as walletWatcher from '../../src/wallet-watcher';
 import * as fixtures from '../fixtures';
-import { ChainsConfig, Wallet } from '../../src/types';
 
 // Jest version 27 has a bug where jest.setTimeout does not work correctly inside describe or test blocks
 // https://github.com/facebook/jest/issues/11607
 jest.setTimeout(60_000);
 
-const oldEnv = process.env;
-
 describe('walletWatcher', () => {
   const config = fixtures.buildConfig();
   const wallets = fixtures.buildWallets();
-  const networks = fixtures.buildNetworks();
-  const chainName = 'localhost';
   const chainId = '31337';
-  const globalSponsorAddress = '0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266';
-  const topUpWalletAddress = '0xC26f10e1b37A1E7A7De266FeF0c19533489C3e75';
-  const provider = new ethers.providers.StaticJsonRpcProvider(config.chains[chainId].rpc, {
-    chainId: parseInt(chainId),
-    name: chainName,
-  });
-  const wallet = {
-    walletType: 'Provider' as WalletType,
-    address: topUpWalletAddress,
-    chainName,
-  };
-
-  let sendToOpsGenieLowLevelSpy: jest.SpyInstance;
-  let closeOpsGenieAlertWithAliasSpy: jest.SpyInstance;
-  let getGasPriceSpy: jest.SpyInstance;
 
   beforeEach(async () => {
-    process.env = oldEnv;
     // Reset the local hardhat network state for each test
     await hre.network.provider.send('hardhat_reset');
-    jest.restoreAllMocks();
-    jest.clearAllTimers();
-
-    // Reset alerts to ensure a clean state for each test
-    opsGenie.resetCachedAlerts();
-    opsGenie.resetOpenAlerts();
-
-    //Spy on getGasPrice
-    getGasPriceSpy = jest.spyOn(nodeUtils, 'getGasPrice');
-
-    // Mock calls to 3rd party APIs
-    sendToOpsGenieLowLevelSpy = jest.spyOn(opsGenie, 'sendToOpsGenieLowLevel').mockImplementation(async () => {
-      console.log('sendToOpsGenieLowLevel was called');
-      return;
-    });
-    closeOpsGenieAlertWithAliasSpy = jest
-      .spyOn(opsGenie, 'closeOpsGenieAlertWithAlias')
-      .mockImplementation(async () => {
-        console.log('closeOpsGenieAlertWithAlias was called');
-        return;
-      });
-    jest.spyOn(opsGenie, 'getOpenAlertsForAlias').mockImplementation(async () => {
-      console.log('getOpenAlertsForAlias was called');
-      return '' as any;
-    });
-    jest.spyOn(opsGenie, 'listOpenOpsGenieAlerts').mockImplementation(async () => {
-      console.log('listOpenOpsGenieAlerts was called');
-      return '' as any;
-    });
 
-    // Mock airnode-protocol networks
-    jest.spyOn(walletWatcher, 'getNetworks').mockImplementationOnce(() => networks);
+    jest.clearAllMocks();
   });
 
   describe('runWalletWatcher', () => {
-    it('tops up wallet succesfully if WALLET_ENABLE_SEND_FUNDS is enabled', async () => {
-      process.env.WALLET_ENABLE_SEND_FUNDS = 'true';
-      const nonceManagerSendTransactionSpy = jest.spyOn(ethersExperimental.NonceManager.prototype, 'sendTransaction');
+    it('sends alert to OpsGenie when wallet is below threshold', async () => {
```

Comment:
e2e now does not make much sense since there is no state change in the blockchain being triggered. I just left a happy path test but it is too similar to the current unit tests 🤷🏻